Src/Base/AMReX_omp_mod.F90

```diff
@@ -4,7 +4,7 @@ module amrex_omp_module
 
   implicit none
 
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
   integer, parameter :: amrex_omp_support = (_OPENMP)
```

Comment:
Curious crash - seems _OPENMP is not defined even with `BL_USE_OMP`/`AMREX_USE_OMP` being defined

Reply:
maybe `-fopenmp` is missing.

Reply:
Yes, it definitely is missing for the Fortran part, let's add it.